Lecture6(GithubAPI)/components/HeaderComponent.js

```diff
@@ -1,16 +1,42 @@
 import { SocialMediaIcon as icon } from "../utils/socialMediaIcon";
-const HeaderComponent = () => (
-  <nav className="navbar navbar-light bg-light">
-    <a className="navbar-brand" href="#">
-      <img
-        className="d-inline-block align-top mr-4"
-        width={50}
-        src={icon.logo}
-        alt="logo"
-      />
-      {icon.title}
-    </a>
-  </nav>
-);
+import { Link, NavLink } from "react-router-dom";
+import { useContext } from "react";
+import ThemeContext from "../context/ThemeContext";
+
+const HeaderComponent = () => {
+  const { themeContext, setThemeContext } = useContext(ThemeContext);
+
+  let activeStyle = {
+    textDecoration: "underline",
+  };
+
+
+  return (
+    <nav className={`navbar ${themeContext === "dark" ? "navbar-dark bg-dark" : "navbar-light bg-light"}`}>
+      <Link className="navbar-brand nav-link" href="#">
+        <img
+          className="d-inline-block align-top mr-4"
+          width={40}
+          src={icon.logo}
+          alt="logo"
+        />
+        <span className="showTitle">{icon.title}</span>
+      </Link>
+      <div className="tabRight">
+        <NavLink style={({ isActive }) =>
+          isActive ? activeStyle : undefined
```

Comment:
use null instead of undefined. We use undefined when. there is no value

Reply:
done